content/tutorials/r_citations_markdown/index.Rmd

```diff
@@ -118,7 +118,7 @@ To specify the style file for the Research Institute for Nature and Forest it is
                     csl = "https://raw.githubusercontent.com/citation-style-language/styles/master/research-institute-for-nature-and-forest.csl")
     ```
 
-### Other YAML citation arguments {#other-yaml-citation-arguments}
+### Other YAML citation arguments {\#other-yaml-citation-arguments}
```

Comment:
I suggest to revert this change, it doesn't follow the recommended markdown syntax.